### PR TITLE
Remove brandedFrontBody template

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
[brandedFrontBody](https://github.com/guardian/frontend/blob/b233fe9/facia/app/views/fragments/brandedFrontBody.scala.htmll) and [FrontBody](https://github.com/guardian/frontend/blob/b233fe9/facia/app/views/fragments/frontBody.scala.html) templates are almost identical.
Removing brandedFrontBody and moving the tiny part of logic that is
different into the PressedPage class

## What is the value of this and can you measure success?
Less duplicate code and indirections
Less code = Less bugs

## Tested in CODE?
Not Yet

### CODE REVIEW

**Review by NataliaLKB — APPROVED** [review]

**Comment by NataliaLKB** [comment]
Changes look good to me! 

cc @guardian/commercial-dev just in case you have any objections.

**Comment by rich-nguyen** [comment]
that looks functionally equivalent, so 👍 from me

File: common/app/model/PressedPage.scala [hunk 1]
```diff
@@ -101,12 +102,22 @@ case class PressedPage (
...
 
   def allItems = collections.flatMap(_.curatedPlusBackfillDeduplicated).distinct
 
-  def isSponsored(edition: Edition): Boolean = metadata.commercial.exists(_.isSponsored(edition))
-  val isPaid: Boolean = metadata.commercial.exists(_.isPaidContent)
+  def isBranded(edition: Edition): Boolean = branding(edition).isDefined
+
+  def isSponsored(edition: Edition): Boolean = branding(edition)
+    .map(_.isSponsored)
+    .getOrElse(metadata.commercial.exists(_.isSponsored(edition)))
```
Comment (kelvin-chappell) [inline common/app/model/PressedPage.scala]: Isn't this the same condition twice?

File: common/app/model/PressedPage.scala [hunk 2]
```diff
@@ -101,12 +102,22 @@ case class PressedPage (
...
+
+  def isPaid(edition: Edition): Boolean = branding(edition)
+    .map(_.isPaid)
+    .getOrElse(metadata.commercial.exists(_.isPaidContent))
+
```
Comment (kelvin-chappell) [inline common/app/model/PressedPage.scala]: And same here.
Could just be `branding(edition).map(_.isPaid)`

**Review by kelvin-chappell — COMMENTED** [review]
I can't think why we didn't do this before.  It must have slipped through the gaps.
Just a couple of comments.

File: common/app/model/PressedPage.scala (hunk 1 above)
Comment (TBonnin) [inline common/app/model/PressedPage.scala]: You are right. It is the same condition twice. Will remove one

File: common/app/model/PressedPage.scala (hunk 2 above)
Comment (TBonnin) [inline common/app/model/PressedPage.scala]: you are right again. will replace by `branding(edition).exists(_.isPaid)`

**Comment by TBonnin** [comment]
@kelvin-chappell I implemented the changes you had recommended.

**Review by kelvin-chappell — APPROVED** [review]
👍

**Pull request merged**